app/Jobs/Migrations/ProcessIntegrationPage.php

```
@@ -69,6 +70,200 @@ public function handle(): void
                 return;
             }
 
+            if ($service === 'monzo') {
+                $pluginClass = PluginRegistry::getPlugin('monzo');
+                $plugin = new $pluginClass();
+                $type = $this->context['instance_type'] ?? 'transactions';
+                $processingPhase = (bool) ($this->context['processing_phase'] ?? false);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$processingPhase'. (Unused Code Rules)

(UnusedLocalVariable)